src/test/kotlin/com/github/epadronu/balin/core/BrowserSpec.kt

```diff
@@ -16,285 +16,306 @@
 
 /* ***************************************************************************/
 package com.github.epadronu.balin.core
+
 /* ***************************************************************************/
 
 /* ***************************************************************************/
```

Comment:
Use this separators in the files you created. I know it is a personal taste but it makes the different sections of a Kotlin file more explicit and clear.